### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by dlharmon** [ai03-2725/Type-C.pretty#2]
I'd appreciate it if you would add a license file to this repo. Thanks.

**Comment by evyd13** [comment]
Yeah I'm not sure if I can share step files of my pcbs with these connectors on them

**Comment by jenschr** [comment]
Yeah same for me. Someone contributed a modified step, but I don't know if I can redistribute it with my own files. Mine use the unlicensed so as many as possible can use the files https://github.com/jenschr/USB-C-Connectors/blob/master/LICENSE
